.github/BOTMETA.yml

```diff
@@ -875,6 +835,93 @@ files:
     - core inventory
     - inventory
     - inventory parsing
+  $module_utils/a10.py: ericchou1 mischapeters
+    labels:
+    - networking
```

Comment:
this is invalid YAML, once you expand the entry into a dict, you cannot have 'inline maintainers', they need to go into their own key.